### PR TITLE
Change bridge function: Make the NIC appear in the "Local Bridge Settings" list

### PR DESCRIPTION
**Pull request by hiura2023**
Changes proposed in this pull request:
 -  I can not view the NIC with this version of VPN Server.
    NIC does not appear in the "Local Bridge Settings" list, If variable "FriendlyName" consists of a NULL character.

 -  Changes make the NIC appear in the "Local Bridge Settings" list regardless of a NULL character consisted in "FriendlyName"
    Modification on combination name below. Beginning space is deleted.
     ①from "FriendlyName"+" (ID=9999999999)" to "FriendlyName"+"(ID=9999999999)" 
     ②from "FriendlyName"+" (9...)" to "FriendlyName"+"(9...)" 
     ③from "FriendlyName"+" (Microsoft's Packet Scheduler)" to "FriendlyName"+"(Microsoft's Packet Scheduler)"

    "FriendlyName" is a variable name used in source code.

### CODE REVIEW

File: src/Cedar/BridgeWin32.c [hunk 1]
```diff
@@ -1161,7 +1161,9 @@ void Win32EthMakeCombinedName(char *dst, UINT dst_size, char *nicname, char *gui
 
 	if (IsEmptyStr(guid) == false)
 	{
-		Format(dst, dst_size, "%s (ID=%010u)", nicname, Win32EthGenIdFromGuid(guid));
+		// Allow to combine "FriendlyName" consisting of a NULL character and ID.
+		//Format(dst, dst_size, "%s (ID=%010u)", nicname, Win32EthGenIdFromGuid(guid));
```
Comment (metalefty) [inline src/Cedar/BridgeWin32.c:1165]: What is the reason for leaving old code in comments?  The old code is kept in git history and I don't see any good reason to do such thing everywhere.

**Review by metalefty — CHANGES_REQUESTED** [review]

File: src/Cedar/BridgeWin32.c (hunk 1 above)
Comment (chipitsine) [reply src/Cedar/BridgeWin32.c:1165]: personally I prefer to remove comments.
also `//` is C++ standard while source is C


but honestly, I think it is cosmetic (and we can cut comments in follow up PR for example :)

**Comment by chipitsine** [comment]
@hiura2023 , this `NIC does not appear in the "Local Bridge Settings" list, If variable "FriendlyName" consists of a NULL character.` sounds like a bug fix, right ? it is not a new feature ?

**Comment by metalefty** [comment]
I remember some people complained about some NICs doesn't appear on the bridge configuration but am not sure if he intends to address that issue in this PR.

**Comment by metalefty** [comment]
> I can not view the NIC with this version of VPN Server.

Also, if it's a bugfix, we need to clarify what kind of NIC, on which platform, which version before modifying the code.

**Comment by chipitsine** [comment]
it would be nice to try the same with SE Stable (maybe fix should be propagated to stable branch as well)

`I can not view the NIC with this version of VPN Server.`

File: src/Cedar/BridgeWin32.c (hunk 1 above)
Comment (hiura2023) [reply src/Cedar/BridgeWin32.c:1165]: >everywhere ?
It is false.
Are you a board member of this repository ?

Comment (metalefty) [reply src/Cedar/BridgeWin32.c:1165]: I don't get the point of the question.

Comment (hiura2023) [reply src/Cedar/BridgeWin32.c:1165]: BOARD MEMBERS OF THIS REPOSITORY is described here

https://github.com/SoftEtherVPN/SoftEtherVPN?tab=readme-ov-file#board-members-of-this-repository


Your name not found here. So, I guess that you are not a board member of this repository.

You should say your preference to the BOARD MEMBERS OF THIS REPOSITORY not me.

**Comment by hiura2023** [comment]
>NIC does not appear in the "Local Bridge Settings" list, If variable "FriendlyName" consists of a NULL character. sounds like a bug fix, right ? it is not a new feature ?


I think that it is a bug in Vpn Server or SeLow driver or NIC driver.

File: src/Cedar/BridgeWin32.c (hunk 1 above)
Comment (chipitsine) [reply src/Cedar/BridgeWin32.c:1165]: @hiura2023 , it is easy to find out the list of board members, why are you asking @metalefty whether he is a board member or not ?

**Comment by hiura2023** [comment]
- Execution result after applying the first change.
Attached A
- Execution result after applying the first and second changes.
Attached B

**Attached A**
BRIDGE_NIC_REVA_202403010

**Attached B**
BRIDGE_NIC_REVB_202403010

**Comment by metalefty** [comment]
I think we need to know more context of this PR, which is not
explained enough..

As far as I understand, the main topic of this PR is to sync
SuGetAdapterList function to stable version. The code of the function
comes from v4.03-9408-rtm. Some bugfixes are made in stable edition but
it is not imported to developer version. Stable version is ahead of
developer edition on this point.

Then I hope we can understand that he trying to do in this PR.

**Comment by chipitsine** [comment]
indeed, stable edition has similar change

https://github.com/SoftEtherVPN/SoftEtherVPN_Stable/blob/master/src/Cedar/SeLowUser.c#L849

**Comment by chipitsine** [comment]
I searched through issues, nothing comes to mind, I think we can merge it

**Pull request merged**